modules/administration/pages/maintenance-window.adoc

```diff
@@ -0,0 +1,17 @@
+[[maintenance-window]]
+= Maintenance Window Checklist
+
+If you work on a system with scheduled maintenance windows, you might find it difficult to remember all the things that you need to during that critical downtime.
+
+This section contains a checklist for your maintenance window, with links to further information on performing each of the steps, if you need it.
+
+
+. Stop services.
+. Check if the configuration is still correct, and apply the highstate.
+. Apply the latest updates.
+See xref:upgrade:server-upgrade.adoc[].
+. Update to the latest service pack, if required.
+See xref:upgrade:sp-migration.adoc[].
+. Reboot the server.
```

Comment:
That's "safe", but not always necessary. You can even update (parts of) the kernel with live patching.

Reply:
Agreed, but it's probably best to mention it.